api/src/main/java/com/cloud/vm/UserVmService.java

```diff
@@ -425,6 +425,8 @@ UserVm createVirtualMachine(DeployVMCmd cmd) throws InsufficientCapacityExceptio
 
     UserVm getUserVm(long vmId);
 
+    VirtualMachine getVm(long vmId);
```

Comment:
It does, however, to make this happen it caused a circular dependency problem, and to solve it, it would need to refactor a lot of classes to fix the issue. The refactor could be interesting in another PR, but is out of the scope of this one.

Reply:
okay @BryanMLima